app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,19 @@
+class MerchantsController < ApplicationController
+
+  def new
+  end
+
+  def index
+    @merchants = MerchantFacade.find_merchants
+    #require "pry"; binding.pry
+  end
+
+  def show
+    @merchant = MerchantFacade.find_one_merchant(params[:id].to_i)
+
+    items_response = Faraday.get("http://localhost:3000/api/v1/merchants/#{params[:id].to_i}/items")
```

Comment:
This needs to be done in your service, not your controller. Just like your `find_one_merchant` method above calls to the service to get the data it needs, you should do the same thing here for a merchants items.